slack-base/src/main/java/com/hubspot/slack/client/models/response/channels/ChannelsKickResponseIF.java

```diff
@@ -0,0 +1,14 @@
+package com.hubspot.slack.client.models.response.channels;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.response.SlackResponse;
+
+@HubSpotStyle
+@Value.Immutable
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ChannelsKickResponseIF extends SlackResponse {
```

Comment:
according to https://api.slack.com/methods/channels.kick there's no more information than the base slack response.